parts/kuberneteskubelet.service

```diff
@@ -45,6 +45,7 @@ ExecStart=/usr/bin/docker run \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
         --hairpin-mode=promiscuous-bridge \
         --network-plugin=${KUBELET_NETWORK_PLUGIN} \
+        --feature-gates=Accelerators=true \
```

Comment:
For my own edification, does the templating engine support wrapping manifests like this one with golang conditionals, e.g. 

```
{{if ge .OrchestratorProfile.OrchestratorVersion 1.6.0}}
--feature-gates=Accelerators=true \
{{end}}
```